src/breadcrumb/src/BreadcrumbItem.tsx

```diff
@@ -1,9 +1,18 @@
-import { h, defineComponent, inject, ExtractPropTypes } from 'vue'
+import {
+  h,
+  defineComponent,
+  inject,
+  ExtractPropTypes,
+  PropType,
+  computed
+} from 'vue'
 import { warn } from '../../_utils'
+import { useBrowserLocation } from '../../_utils/composable/use-browser-location'
 import { breadcrumbInjectionKey } from './Breadcrumb'
 
 const breadcrumbItemProps = {
-  separator: String
+  separator: String,
+  href: String as PropType<string | undefined>
```

Comment:
```suggestion
  href: String
```

`PropType` is not needed, the inferred type would be the same.